src/RpcNep17Tracker/Nep17Balance.cs

```diff
@@ -6,20 +6,21 @@ namespace Neo.Plugins
 {
     public class Nep17Balance : ISerializable
     {
-        public BigInteger Balance;
+        public BigDecimal Balance;
         public uint LastUpdatedBlock;
 
-        int ISerializable.Size => Balance.ToByteArray().GetVarSize() + sizeof(uint);
+        int ISerializable.Size => Balance.Value.ToByteArray().GetVarSize() + sizeof(byte) + sizeof(uint);
 
         void ISerializable.Serialize(BinaryWriter writer)
         {
-            writer.WriteVarBytes(Balance.ToByteArray());
+            writer.WriteVarBytes(Balance.Value.ToByteArray());
+            writer.Write(Balance.Decimals);
             writer.Write(LastUpdatedBlock);
         }
 
         void ISerializable.Deserialize(BinaryReader reader)
         {
-            Balance = new BigInteger(reader.ReadVarBytes(512));
+            Balance = new BigDecimal(new BigInteger(reader.ReadVarBytes(512)), reader.ReadByte());
```

Comment:
The order of arguments calculation is non-deterministic.

Reply:
Done